src/aavrutiDJun20/ValidatePinNumber.java

```diff
@@ -0,0 +1,62 @@
+package aavrutiDJun20;
+
+import java.util.Arrays;
+
+public class ValidatePinNumber {
+	
+	void validatePin(String number) {
+		String num = "";
+		boolean flag = true;
+		
+		try {
+			Integer.parseInt(number);
+			
+			for(int index=0;index<number.length();index++) {
+				if(num.contains(String.valueOf(number.charAt(index)))) {
+					flag = false;
+					break;
+				}
+				else {
+					num += number.charAt(index);
+				}
+			}
+			if(flag) {
+				char charArray[] = number.toCharArray();
+				Arrays.sort(charArray);
+				String numSorted = String.valueOf(charArray);
+				
+				StringBuilder strReverse = new StringBuilder(numSorted);
+				strReverse.reverse();
```

Comment:
Hi @AavrutiDJun20 
I feel using this reverse() method we not achieving our goal, Here we need to check ascending and descending order, not reverse order. check once